### PR TITLE
LOC CHECKIN | visualfsharp master | 20180202

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/fsharp/xlf/FSStrings.de.xlf [hunk 1]
```diff
@@ -1284,27 +1284,27 @@
...
         <note />
       </trans-unit>
       <trans-unit id="UnitTypeExpectedWithEquality">
         <source>The result of this equality expression has type '{0}' and is implicitly discarded. Consider using 'let' to bind the result to a name, e.g. 'let result = expression'.</source>
-        <target state="needs-review-translation">Das Ergebnis dieses Gleichheitsausdrucks wird implizit verworfen. Verwenden Sie ggf. "let", um das Ergebnis an einen Namen zu binden. Beispiel: "let Ergebnis = Ausdruck".</target>
+        <target state="translated">Das Ergebnis dieses Gleichheitsausdrucks weist den Typ "{0}" auf und wird implizit verworfen. Verwenden Sie ggf. "let", um das Ergebnis an einen Namen zu binden. Beispiel: "let Ergebnis = Ausdruck".</target>
         <note />
       </trans-unit>
       <trans-unit id="UnitTypeExpectedWithPossiblePropertySetter">
         <source>The result of this equality expression has type '{0}' and is implicitly discarded. Consider using 'let' to bind the result to a name, e.g. 'let result = expression'. If you intended to set a value to a property, then use the '&lt;-' operator e.g. '{1}.{2} &lt;- expression'.</source>
```
Comment (forki) [inline src/fsharp/xlf/FSStrings.de.xlf]: ok quick q: how can a "equality expression" have any other type than bool? /cc @dsyme @isaacabraham

**Comment by brettfo** [comment]
@dotnet-bot test Ubuntu14.04 Release_default Build please

File: src/fsharp/xlf/FSStrings.de.xlf (hunk 1 above)
Comment (dsyme) [reply src/fsharp/xlf/FSStrings.de.xlf]: I suppose you could redefine the equality operator

Comment (forki) [reply src/fsharp/xlf/FSStrings.de.xlf]: Nice one. Also: evil ;-)

**Pull request merged**